app/assets/javascripts/discourse/app/controllers/edit-category-tabs.js

```diff
@@ -115,15 +116,19 @@ export default Controller.extend({
         I18n.t("yes_value"),
         (result) => {
           if (result) {
-            this.model.destroy().then(
-              () => {
-                this.transitionToRoute("discovery.categories");
-              },
-              () => {
-                this.displayErrors([I18n.t("category.delete_error")]);
+            this.model
+              .destroy()
+              .then(
+                () => {
+                  this.transitionToRoute("discovery.categories");
+                },
```

Comment:
Is there a reason you didn't use a `catch` here? I think it reads easier than having two methods separated by a comma.

Reply:
It was already like that, I should have fixed it though. Will do so shortly.